app/src/organisms/ProtocolSetupLabware/index.tsx

```diff
@@ -394,97 +461,85 @@ function RowLabware({
           mod => mod.moduleId === initialLocation.moduleId
         )
       : null
+  const matchingHeaterShaker =
+    matchedModule?.attachedModuleMatch != null &&
+    matchedModule.attachedModuleMatch.moduleType === HEATERSHAKER_MODULE_TYPE
+      ? matchedModule.attachedModuleMatch
+      : null
+
   const moduleInstructions = (
     <StyledText
       color={COLORS.darkBlack70}
-      fontSize={TYPOGRAPHY.fontSize20}
+      fontSize={TYPOGRAPHY.fontSize22}
       fontWeight={TYPOGRAPHY.fontWeightRegular}
       lineHeight={TYPOGRAPHY.lineHeight24}
```

Comment:
```suggestion
      as="label"
```

https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen?type=design&node-id=6994-101164&mode=design&t=FCtN4GoeQsoGl2WK-0